src/screens/editor/container/editorContainer.js

```diff
@@ -103,14 +103,14 @@ class EditorContainer extends Component {
 
         upload.forEach((el) => {
           if (el.filePath && el.fileName) {
-            this.setState({ isUploading: true });
+            // this.setState({ isUploading: true });
```

Comment:
any reason why we commenting this? it should show progress/spinner while upload is happening, doesn't this break that?

Reply:
spinner still kicks in from `_uploadImage` function. Also, for logged out user it was causing spinner to show indefinitely